packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts

```diff
@@ -51,7 +51,7 @@ export interface BaseLoadBalancerProps {
   /**
    * Indicates whether cross-zone load balancing is enabled.
    *
-   * @default - false for Network Load Balancers and true for Application Load Balancers.
+   * @default - false for Network Load Balancers and true for Application Load Balancers (this cannot be changed).
```

Comment:
Need some clarification, I see [here](https://github.com/aws/aws-cdk/issues/29866#issuecomment-2064577629) and [here](https://github.com/aws/aws-cdk/issues/29866#issuecomment-2061508349) that there are way to disable crossZone for ALB. Why are we saying this cannot be changed here? If we can change it then this should be implement the change suggested [here](https://github.com/aws/aws-cdk/issues/29866#issuecomment-2065180727) imo. Let me know if I'm understanding this wrong.

Reply:
@aaythapa In https://github.com/aws/aws-cdk/issues/29866#issuecomment-2064577629 and   https://github.com/aws/aws-cdk/issues/29866#issuecomment-2061508349, it is mentioned that the `crossZone` could be disabled at `Target Group` level per docs and it already works per example. However, it cannot be changed at **ALB level** per docs. This PR is only to make it clear in LoadBalancer docs that it cannot be changed at ALB level.

**NOTE:** I'm unsure what will happen in CloudFormation deployment if we enforce `crossZone` to `false`, when service itself doesn't support disabling it at ALB level.